DOCS.md

```diff
@@ -28,7 +27,7 @@ The following is a sample Docker configuration in your .drone.yml file:
 publish:
   docker:
     username: kevinbacon
-    password: pa55word
+    password: $$DOCKER_PASSWORD
```

Comment:
I vote to not use the `$$` notation in the documentation. It is likely we could change this notation in a future release, or make secrets pluggable, in which case we might not want the notation hard-coded into all the docs. 

I would recommend we instead add a section to the plugin template (so it appears on the all the plugin docs) that recommends using secrets instead of hard-coding into the yaml, and provides a link to the secrets documentation.